plugins/populator/frontend/public/javascripts/countly.models.js

```diff
@@ -410,7 +543,11 @@
                     req["token_session"] = 1;
                     req["test_mode"] = 0;
                     req.events = req.events.concat(this.getPushEvents());
+                    req.events = req.events.concat(this.getHeatmapEvents());
+                    req.events = req.events.concat(this.getFeedbackEvents());
+                    req.events = req.events.concat(this.getScrollmapEvents());
 					req[this.platform.toLowerCase()+"_token"] = randomString(8);
+                    this.createFeedbackWidget();
```

Comment:
Creating widget should be better done outside of user scope, check how attribution creates campaigns